packages/browser/src/Session.ts

```diff
@@ -179,7 +193,8 @@ export class Session extends EventEmitter {
    * using the browser's current location.
    */
   handleIncomingRedirect = async (
-    url: string = window.location.href
+    url: string = window.location.href,
+    options: IHandleIncomgingRedirectOptions = {}
```

Comment:
Actually, I'm thinking, now that `url` is optional, maybe we should make `options` the first argument and have `url` be one of its parameters (and allow it to be a string for backwards compatibility)?

Reply:
Now added this as 57de44c7 - let me know what you think.